packages/target-nodejs-sdk/demo/package.json

```diff
@@ -10,7 +10,10 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
```

Comment:
we should probably also add `"@mutagen-d/node-proxy-server"` to dependencies

Reply:
I am not sure if it is necessary to include the local proxy server file for this PR